pop-cli-for-smart-contracts/guides/calling-your-contract.md

```diff
@@ -2,14 +2,30 @@
 
 Now that your contract is deployed locally, we can start interacting with it.
 
-Grab the contract address that was outputted from the [previous step](deploy-your-contract-locally/deploy-locally-on-a-solochain.md).
+### Interactive Guidance (Recommended)
+
+Interact with your deployed contract **with** Pop CLI's interactive guidance by simply entering:
```

Comment:
We need to keep consistency throughout the docs; otherwise, it’ll turn into a mess. 
I followed the same style as before: https://learn.onpop.io/appchains/guides/create-a-new-parachain